src/js/angular/export/controllers.js

```diff
@@ -207,6 +217,29 @@ exportCtrl.controller('ExportCtrl',
                 }
             };
 
+          /**
+           * Calls the GET graphs endpoint without params, in order to download all the graphs from the BE.
+           */
+          $scope.downloadAllGraphs = function () {
+              if ($scope.getActiveRepository()) {
+                  RDF4JRepositoriesRestService.getGraphs($scope.getActiveRepository())
+                      .then(function (res) {
+                          FileUtils.downloadAsFile(fetchDownloadAllGraphsFilename(res), res.headers()['content-type'], JSON.stringify(res, null, '\t'));
```

Comment:
This should be in the service as well as the fetchDownloadAllGraphsFilename function

Reply:
Also, you can use the FileSaver (saveAs) library for this.

Reply:
Why you stringify the response?

Reply:
Refactored